scipy/linalg/blas.py

```diff
@@ -214,7 +214,10 @@ def find_best_blas_type(arrays=(), dtype=None):
             prefer_fortran = True
 
     prefix = _type_conv.get(dtype.char, 'd')
-    if dtype.char not in 'fdFD':
```

Comment:
This seems to be the case already: https://github.com/scipy/scipy/pull/6207/files#diff-51d8d45efc9d4bd7634e0d31ee815602L167